index.js

```diff
@@ -129,6 +160,38 @@
       Object.prototype.toString.call(x).slice('[object '.length, -']'.length);
   }
 
+  //# type.parse :: String -> { namespace :: Nullable String, name :: String, version :: Number }
+  //.
+  //. Takes any String and parses it according to the [specification][3],
+  //. returning an object with a `namespace`, a `name` and a `version` field.
+  //.
+  //. ```javascript
+  //. > type.parse('my-package/List@2')
+  //. {namespace: 'my-package', name: 'List', version: 2}
+  //.
+  //. > type.parse('nonsense!')
+  //. {namespace: null, name: 'nonsense!', version: 0}
+  //.
+  //. > type.parse(Identity['@@type'])
+  //. {namespace: 'my-package', name: 'Identity', version: 1}
+  //. ```
+  /* eslint-disable key-spacing */
```

Comment:
> We want to break the rule, because it looks nice in this case.

I'd go further by claiming that in this case it makes the code easier to comprehend.

> Disabling the rule altogether would open the rest of the code up for this mistake.

I like having strict rules as a starting point, but I'm happy to break the rules in some cases. The makefiles for the various Sanctuary projects disable several rules. Let's add `--rule 'key-spacing: [off]'` to the makefile.